tests/unit/test_utils.py

```diff
@@ -3,12 +3,10 @@
 
 from scanapi.errors import InvalidKeyError, MissingMandatoryKeyError
 from scanapi.utils import (
```

Comment:
I would suggest we could split the test file too, the same way you did for the utils and hide_utils. What do you think?

Reply:
Good idea! I'll do it